### DISCUSSION TITLE
Server Query for external servers in Server Block

### DISCUSSION THREAD

**Discussion by ImpulsiveFPS** [gameserverapp/Platform#377] [Suggestions]
It would be a nice feature to be able to add external servers by server query to show them in server blocks on your website.
For those servers that are not supported but you still want them to show up as servers on your website.

Right now I am embedding a image from Battlemetrics instead of using the server block because I want all servers listed to show the same information.

If we could add external servers to be used in the Server Block this would look way more clean. 

Might be niche use case but as many communities host games for multiple games some supported but others that are not this would make GSA website functionality a lot more versatile.

**Comment by mvaessen** [comment]
Hi, we do support that through RconConnect. If you want to add servers that arent supported on GSA, i recommend using the self-hosted community website and program the extra servers that way

**Reply by ImpulsiveFPS** [reply]
Even if the server does not have Rcon Support?
Ah I found the self-hosted one. I guess this is more in line what I actually want! Thanks!

**Reply by mvaessen** [reply]
Sure, you can build anything on the self-hosted website. You probably do need some experience with coding, so if you have that it should be fine.

For the suggestion though, i currently have no plans to support servers that aren't connected to GSA. So for people that want this, I recommend using code to change the self-hosted community website.

**Reply by ImpulsiveFPS** [reply]
> Sure, you can build anything on the self-hosted website. You probably do need some experience with coding, so if you have that it should be fine.
> 
> For the suggestion though, i currently have no plans to support servers that aren't connected to GSA. So for people that want this, I recommend using code to change the self-hosted community website.

This sounds good enough for me! I have some experience so I should be able to manage that :)
